test/test.ts

```diff
@@ -17,79 +13,58 @@ import {
 
 // Tests a UDP identify flow end-to-end
 test('udp-device-connects', async (t) => {
+  // Common address for all UDP messaging
+  const updAddress: string = '255.255.255.255';
```

Comment:
If this can be a common variable, can it be made a global variable in this file?

Reply:
Done.